<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>400</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="start_page">
     <widget class="QLabel" name="intro">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>511</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Thank you for participating in this experiment. This experiment will have you complete a few simple tasks on this computer. This experiment does not track your actions or steal any personal information. There is minimal risk involved in participating in this experiment. This experiment will take about 20 minutes to complete.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="consent_btn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>I consent</string>
      </property>
     </widget>
     <widget class="QLabel" name="outline">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>511</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>This experiment is designed to test how computer users respond to some computer system modifications we are prototyping. We will first have you complete some simple tasks with these modifications turned off. Later, we will turn on these modifications and ask you to complete the same set of tasks.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="consent">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Do you consent to participate in this study? You may exit the experiment at any point.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="not_consent_btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>I do not consent</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="consent_page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>81</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="goodbye_page"/>
    <widget class="QWidget" name="task1_page">
     <property name="styleSheet">
      <string notr="true">background-color:blue</string>
     </property>
    </widget>
    <widget class="QWidget" name="patch_page">
     <property name="styleSheet">
      <string notr="true">background-color:red</string>
     </property>
    </widget>
    <widget class="QWidget" name="task2_page"/>
    <widget class="QWidget" name="q1_page"/>
    <widget class="QWidget" name="q2_page"/>
    <widget class="QWidget" name="q3_page"/>
    <widget class="QWidget" name="q4_page"/>
    <widget class="QWidget" name="wta_page"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
